sdk/keyvault/azkeys/internal/generated/keyvault_client.go

```diff
@@ -502,9 +502,9 @@ func (client *KeyVaultClient) getKeyCreateRequest(ctx context.Context, vaultBase
 		return nil, errors.New("parameter keyName cannot be empty")
 	}
 	urlPath = strings.ReplaceAll(urlPath, "{key-name}", url.PathEscape(keyName))
-	if keyVersion == "" {
-		return nil, errors.New("parameter keyVersion cannot be empty")
-	}
+	// if keyVersion == "" {
```

Comment:
Do you know if there's something in the swagger so that we know to omit this check?

Reply:
Hmm it's marked as required but the description says it's optional.

https://github.com/Azure/azure-rest-api-specs/blob/ecdce42924ed0f7e60a32c74bc0eb674ca6d4aae/specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.3-preview/keys.json#L287-L293